### ISSUE TITLE
create-remix@latest does not work with screen reader

### ISSUE THREAD

**Issue by blindgumption** [remix-run/remix#2419]
### What version of Remix are you using?

1.3.2

### Steps to Reproduce

in bash shell, on WSL2/Windows 11/Ubuntu 20.04, run create-remix@latest 
you are told to use arrow keys to select options 
if you're using a screen reader (I'm using NVDA version 3.4.2021) the option text is not read out.  
I had my sighted son try it and he told me the arrows highlight each option in blue.  
The screen reader only reads the text of the first option 

### Expected Behavior

I was thrilled to try remix after hearing about it on The ChangeLog podcast.  And thought this will be great when two twitter comments on remix.run mentioned accessible/accessibility.
And kudos to remix for addressing accessibility in the docs and calling out great resources for devs to reference.
Imagine my deflated response though when I went to run create-remix@latest as described in the try it out page.
It appears to be using some kind of fancy terminal tool to allow colorizing and arrowing around the terminal.
It does not work with a screen reader.
Does it really need to be that fancy?
How about the old fashioned way of entering the letter in [] to select an option?

### Actual Behavior

I guess I sort of addressed this in the previous comments.
My sighted son tried running it and told me the options were highlighted in blue when you pressed an arrow key.
My screen reader only ever read out the first option regardless of which option was highlighted.
Also, after completion, even without my screen reader running (I had turned it off for my son to try running create-remix), when I restarted the screen reader, the prompt was read out in a strange way.  I restarted NVDA again and the prompt still was read incorrectly.  It was like the synthesizer was being fed the prompt in random chunks.

**Comment by chaance** [comment]
Thanks for flagging. I need to do some Windows debugging tomorrow so I'll crank open NVDA and see what's going on.

**Comment by blindgumption** [comment]
thanks @chaance for the update and work around.  
I've done a bit more poking around.

TL;DR - it might be an issue with NVDA as it mostly works with JAWS.  Given there is a good work-around, it might be better to not spend a lot of time on this now, especially if create-remix is getting an overhaul.

The rest of the story -
I cranked up NVDA to DEBUG and went through the process again, there was nothing obvious in the NVDA logs.  I'm not a developer on NVDA though so could have missed something.  I didn't find any exceptions or any other alarming strings in the logs.

I tried the latest JAWS 2022 and it seemed to work okay.  It was very verbose but that could be my settings and the nature of JAWS and the terminal.  Bottom line is though I did get an audible indication of which option was highlighted.  Also, after running create-remix@latest, my prompt when using JAWS sounded fine.
I use the same speech synthesizer with JAWS and NVDA so seems likely NVDA has done something with the terminal.

In general, I much prefer flags on the command line to having to answer questions when running a script.  As long as that's always there, I'm not too worried about how well create-remix works with NVDA.  As you might imagine though, there are plenty of blind software developers with varying workflows. 

I noticed there's a package attached to this issue.  My depth of debugging with Nodejs environments is not great.  I assume there are changes in that package to try with NVDA.  I'll see if I can get that running locally and let you know how it goes.
cheers,
Joel

**Comment by chaance** [comment]
Closing this because I don't think there's anything actionable on our part, and we are rewriting our CLI very soon. I may ping you for direct feedback along the way, if you don't mind!

**Comment by joeldodson** [comment]
Thanks Chance, sounds good.

 

Yes, please feel free to ping me directly on gmail as joeldodson.

 

I’ve been back in Python for a while, not following the JS world.  I’ll try to remember to give remix a go again soon.

 

Cheers,

Joel
